charts/seed-controlplane/charts/kube-apiserver/templates/kube-apiserver.yaml

```
@@ -53,7 +53,11 @@ spec:
         imagePullPolicy: IfNotPresent
         command:
         - /hyperkube
+        {{- if semverCompare "< 1.15" .Values.kubernetesVersion }}
```

Only the cmd aliases were removed in v1.15 and the cmd names are present in the old hyperkube images. We could remove the if statement and directly use the cmd names {kube-apiserver, kube-scheduler, kube-controller-manager, kube-proxy}.

Ref kubernetes/kubernetes#76953

```suggestion
        - kube-apiserver
```

---

Yes, we could do that, however, this would result in a needless restart of all hyperkube components for older clusters. As we don’t gain much out of it (imho, the if-else-statement is not complex) I would not do it now.